README.md

```diff
@@ -59,11 +59,11 @@ First, you need to have Python >=3.7 installed. If you don't, we recommend you t
   brew install pyenv
   ```
 
-  then Python 3.7 (you could check the latest 3.7.x version with `pyenv install --list`):
+  then Python 3.8 (you could check the latest 3.8.x version with `pyenv install --list`):
```

Comment:
Should we say here 3.10, since below we have `pyenv install 3.10.5`?

Reply:
That makes sense, I'll do that.

Reply:
Done.